src/main/java/core/mate/academy/service/impl/BulldozerProducer.java

```diff
@@ -0,0 +1,14 @@
+package core.mate.academy.service.impl;
+
+import core.mate.academy.model.Bulldozer;
+import core.mate.academy.model.Machine;
+import core.mate.academy.service.MachineProducer;
+
+import java.util.List;
+
+public class BulldozerProducer implements MachineProducer<Bulldozer> {
+    @Override
+    public List<Bulldozer> get() {
+        return List.of(new Bulldozer());

```

Comment:
The `get` method should return a list containing 2-3 `Bulldozer` instances as per the task requirements. Consider creating additional `Bulldozer` objects and adding them to the list.